infra/airgap/scripts/generate-sbom.sh

```
@@ -0,0 +1,296 @@
+generate_vuln_report() {
+    local sbom_file="$1"
+    local vuln_file="${sbom_file%.json}-vulns.json"
+
+    log_info "Scanning for vulnerabilities..."
+
+    grype "sbom:${sbom_file}" \
+        -o json \
+        --file "${vuln_file}"
+
+    # Check for critical/high vulnerabilities
+    local critical_count=$(jq '[.matches[] | select(.vulnerability.severity == "Critical")] | length' "${vuln_file}")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)